apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/brevbaker/BrevbakerRequest.kt

```diff
@@ -58,6 +58,7 @@ enum class EtterlatteBrevKode {
     OMS_FOERSTEGANGSVEDTAK_INNVILGELSE_UTFALL,
     OMS_INNVILGELSE_MANUELL,
     OMS_OPPHOER_MANUELL,
+    OMS_REVURDERING_ENDRING,
```

Comment:
Burde det være  `OMS_REVURDERING_INNTEKTSENDRING`? Eller gjør man dette på to forskjellig måter på BP og OMS?

Reply:
OMS_REVURDERING_ENDRING er hovedmalen til alle revurderinger som inneholder resulterer i endring. Selve innholdet blir styrt av saksbehandler manuelt.